src/main/java/school/lemon/changerequest/java/introduction/hw1/Task3.java

```diff
@@ -2,6 +2,6 @@
 
 public class Task3 {
     public static boolean isBetween(int a, int b, int c) {
-        return false;
+        return (a <= b & b <= c) | (a >= b & b >= c);
```

Comment:
Could you please explain why you are using `|` and `&` instead of `||` and `&&`?

Reply:
To evaluate both sides of expression, but yes, probably we can change | to || and leave & untouched.